src/librustc_middle/mir/interpret/allocation.rs

```diff
@@ -27,7 +27,7 @@ pub struct Allocation<Tag = (), Extra = ()> {
     /// at the given offset.
     relocations: Relocations<Tag>,
     /// Denotes which part of this allocation is initialized.
-    undef_mask: UndefMask,
+    uninit_mask: UninitMask,
```

Comment:
So as discussed before, this should become `init_mask: InitMask`. The negation in `UninitMask` is just confusing, and there's no point in renaming it now only to rename it again in a week.

Sorry for the back-and-forth, sometimes it takes some experimentation to figure out good names for things.

Reply:
This will require negating a bunch of stuff that depends on this, right? Or is it sufficient if I just  negate the impl of `InitMask`. I.e.
```
    pub fn set_range(&mut self, start: Size, end: Size, new_state: bool) {
        let len = self.len;
        if end > len {
            self.grow(end - len, new_state);
        }
        self.set_range_inbounds(start, end, new_state); // becomes self.set_range_inbounds(start, end, !new_state); 
    }
```

Reply:
The logic is *already* that `true` is initialized. That's one reason why this is so confusing...

Reply:
Oh wow, lol. Okay then.